Formula/instead.rb

```diff
@@ -1,8 +1,8 @@
 class Instead < Formula
   desc "Interpreter of simple text adventures"
-  homepage "https://instead.syscall.ru/"
-  url "https://github.com/instead-hub/instead/archive/3.3.4.tar.gz"
-  sha256 "6577235e42a22d8f7f628fedc998d718e65da9e1a3cb61f09be6d1cd648fc061"
+  homepage "http://instead.hugeping.ru/"
```

Comment:
We might want to use https://instead-hub.github.io/ here, but that can be in a follow-up so that we don't need to re-run CI.